src/prefect/filesystems.py

```diff
@@ -188,8 +190,15 @@ class RemoteFileSystem(ReadableFileSystem, WritableFileSystem):
     _block_type_name = "Remote File System"
     _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/4CxjycqILlT9S9YchI7o1q/ee62e2089dfceb19072245c62f0c69d2/image12.png?h=250"
 
-    basepath: str
-    settings: dict = Field(default_factory=dict)
+    basepath: str = Field(
+        ...,
+        description="Default local for this block to write to",
+        example="s3://my-bucket/my-folder/",
```

Comment:
Does example show up in UI?

Reply:
Not yet, but the intent is to have examples show as placeholder text in the future. This adds the example tov `fields` in the block schema, so one the UI supports examples it should show up.